vendor/plugins/edu.harvard.dce.paella.singleVideoToggle/singleVideoToggle.js

```diff
@@ -10,40 +11,76 @@ paella.addPlugin(function () {
   return class SingleVideoTogglePlugin extends paella.ButtonPlugin {
     constructor () {
       super ();
+      this._toggleSingleVideoIndex = 0;
+      this._toggleHlsLiveResV1Index = 1;
+      this._toolTipOpts = ["Switch videos", "Change resolution"];
+      this._subClassOpts = ["showViewModeButton", "showMultipleQualitiesPlugin"];
+      this._iconClassOpts = ["icon-presentation-mode","icon-screen"];
+      this._curOptIndex = 0; // iOS single video: toggle 2 videos to show only one
       this._iOSProfile = 'one_big';
       this._masterVideo = null;
       this._toggleIndex = 1; //toggle to presentation when button pressed first time
     }
     getDefaultToolTip () {
-      return base.dictionary.translate("Switch videos");
+      return base.dictionary.translate(this._toolTipOpts[this._getOptIndex()]);
     }
     getAriaLabel() {
-      return base.dictionary.translate("Switch videos");
+      return base.dictionary.translate(this._toolTipOpts[this._getOptIndex()]);
     }
     getAlignment() {
       return 'right';
     }
     getSubclass() {
-      return "showViewModeButton";
+      // #DCE OPC-497 show current res text on button (get the mutli quality button style)
+      return base.dictionary.translate(this._subClassOpts[this._getOptIndex()]);
     }
     getIconClass() {
-      return 'icon-presentation-mode';
+      // #DCE OPC-497 show current res text on button
+      return base.dictionary.translate(this._iconClassOpts[this._getOptIndex()]);
     }
     getIndex() {
       return 450;
```

Comment:
Just curious. What is 450?

Reply:
@phoebem it's the tab index. Its just to make the tab index happen after some plugins and after others